protocol-dashboard/.env.prod

```diff
@@ -24,4 +24,5 @@ VITE_SOLANA_TOKEN_PROGRAM_ADDRESS=TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA
 VITE_SOLANA_WEB3_CLUSTER=mainnet-beta
 VITE_USDC_MINT_ADDRESS=EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v
 VITE_WAUDIO_MINT_ADDRESS=9LzCMqDgTKYz9Drzqnpgee3SGa89up3a247ypMj2xrqM
-VITE_WORMHOLE_ADDRESS=0x6E7a1F7339bbB62b23D44797b63e4258d283E095
+VITE_WEB3MODAL_PROJECT_ID=0416e7e9c027fb75dc5a365384683fdb
```

Comment:
does it matter that this is public? why do they even have app keys

Reply:
yeah it's not meant to be a secret or anything. i think they have some feature to verify your domain as safe? https://docs.walletconnect.com/cloud/verify
anyway it's fine that this is public bc you can specify an allowlist of domains that can use the key (i put the Audius ones for now, but this means the dashboard instances on individual discovery nodes won't have wallet connect enabled)